package.json

```diff
@@ -74,6 +74,7 @@
     "minimist": "^1.2.0",
     "ncp": "^2.0.0",
     "node-stream-zip": "^1.8.2",
+    "npm-run-all": "^4.1.5",
```

Comment:
can we avoid adding it? this is not a slim dependency: https://npm.anvaka.com/#/view/2d/npm-run-all

I don't think it gives us much. Is it just to shortcut some letters?